spec/board_spec.rb

```diff
@@ -0,0 +1,62 @@
+require './lib/board' 
+
+RSpec.describe Board do
+  describe "initialize" do
```

Comment:
Many time you'll want to use the same object for multiple test case. For that use [`let or let!`](http://www.betterspecs.org/#let)

Reply:
Hint: you can use let at least for an instance of `Board`

Reply:
I honsetly could not understand how to use let, all the examples have some passing methods, to increase count for example. I've used before(:each), but I've read it's worse than let. Could you please write an example of declaring board with let?